### PR TITLE
[v3.6.0] fix deferred pipeline has no shadow && merge error [shadow biase, spotLight.aspect]

### PR DESCRIPTION
**Pull request by troublemaker52025**
Re: #

### Changelog

* 

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.

  > Manual trigger with `@cocos-robot run test cases` afterward.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by jk20012001** [comment]
We need deferred shading pass always enable shadow. Because it's not convenient to distinguish different light sources
and it's hard to get shadow enabled status because scene data is loaded after pipeline initialization.

**Review by jk20012001 — APPROVED** [review]

**Comment by troublemaker52025** [comment]
@cocos-robot run test cases

File: cocos/core/pipeline/custom/ubos.ts [hunk 1]
```diff
@@ -360,7 +360,7 @@ export class PipelineUBO {
             Mat4.invert(_matShadowView, (light as any).node.getWorldMatrix());
             Mat4.toArray(sv, _matShadowView, UBOShadow.MAT_LIGHT_VIEW_OFFSET);
 
-            Mat4.perspective(_matShadowProj, (light as any).angle, (light as any).aspect, 0.001, (light as any).range);
+            Mat4.perspective(_matShadowProj, (light as any).angle, 1.0, 0.001, (light as any).range);
```
Comment (troublemaker52025) [inline cocos/core/pipeline/custom/ubos.ts:363]: v3.5.1 合并到 v3.6.0 解决冲突时被意外丢弃的修改

File: native/tools/tojs/scene.ini [hunk 2]
```diff
@@ -106,7 +106,7 @@ getter_setter = Root::[device:_device mainWindow curWindow tempWindow windows sc
...
+       Model::[scene _subModels:subModels/getSubModels inited/isInited _localDataUpdated/isLocalDataUpdated/setLocalDataUpdated _worldBounds:worldBounds/getWorldBounds/setWorldBounds _modelBounds:modelBounds/getModelBounds/setModelBounds worldBoundBuffer localBuffer updateStamp isInstancingEnabled/isInstancingEnabled receiveShadow/isReceiveShadow/setReceiveShadow castShadow/isCastShadow/setCastShadow shadowBias shadowNormalBias node transform visFlags enabled/isEnabled/setEnabled type instancedAttributes/getInstancedAttributeBlock/setInstancedAttributeBlock isDynamicBatching/isDynamicBatching/setDynamicBatching priority],
```
Comment (troublemaker52025) [inline native/tools/tojs/scene.ini:109]: 模型级别的 shadowBias 及 shadowNormal 也是在版本合并时 被意外的丢弃了

File: cocos/core/pipeline/custom/ubos.ts (hunk 1 above)
Comment (pandamicro) [reply cocos/core/pipeline/custom/ubos.ts:363]: Please start to use English in PR comment

**Review by pandamicro — APPROVED** [review]

**Pull request merged**